osm2lanes/src/transform/tags_to_lanes/counts.rs

```diff
@@ -232,9 +232,9 @@ impl LanesDirectionScheme {
             })
             .map(|_v| {});
         Self {
-            total: tags.get_parsed(LANES, warnings),
-            forward: tags.get_parsed(LANES + "forward", warnings),
-            backward: tags.get_parsed(LANES + "backward", warnings),
+            total: tags.get_parsed(&LANES, warnings),
```

Comment:
This seems like a slight regression in API niceness, but not by much. Working off limited brain capacity, it looks like this used to do some more cloning maybe

Reply:
yeah, maybe we can find a way to add it back later, but I really wrestled with getting this working nicely.
the interface also makes more sense this way (a reference, no internal cloning).